tests/dump_tests/module_tests/mock_sonicv2connector.py

```diff
@@ -0,0 +1,69 @@
+import json, re, os
+from utilities_common.constants import DEFAULT_NAMESPACE
+
+class MockSonicV2Connector():
+    def __init__(self, dedicated_dbs, **kwargs):
+        if "namespace" in kwargs and kwargs["namespace"] != DEFAULT_NAMESPACE:
+            raise "This Mock doesn't support multi-asic configuration"
```

Comment:
Any reason why this mock class doesnt support multi-asic ?

Reply:
Multi-asic tests are already covered in the match_infra tests [LINK](https://github.com/Azure/sonic-utilities/blob/master/tests/dump_tests/match_engine_test.py#L217) and also in the tests which verify the cli output of "dump state" command. There tests are in the next PR [LINK](https://github.com/Azure/sonic-utilities/pull/1668/files#diff-5e77054ec3008dabf912e192441095f5ddf05a1e463be929c7535ff8ca280993R184).

Hence these module specific tests and thus the implementers won't have to bother about testing muti-asic functionality. and thus didn't choose to test the multi-asic related stuff here.